modules/coin_flip.ts

```diff
@@ -20,11 +20,11 @@ export const commands = [
 ]
 
 // returns a random "Heads" or "Tails"
-function flipC(): string {
+function flipC (): string {
```

Comment:
why?

Reply:
Standardjs ruling: https://eslint.org/docs/rules/space-before-function-paren

Reply:
I really don't like this... but since it's the standard I don't really have any say.

IMO it should be this:

```javascript
function foo() {}
bar = function () {}
baz = function quux() {}
```

Reply:
`{"anonymous": "always", "named": "never", "asyncArrow": "always"}` option basically